pdb2sql/StructureSimilarity.py

```diff
@@ -666,9 +672,12 @@ def compute_irmsd_pdb2sql(self,cutoff=10,method='svd',izone=None,exportpath=None
         if izone is None:
             contact_ref = sql_ref.get_contact_atoms(
                 cutoff=cutoff,
-                extend_to_residue=True,
-                return_only_backbone_atoms=True)
-            index_contact_ref = contact_ref['A'] + contact_ref['B']
+                extend_to_residue=True)
+            index_contact_ref = []
+            for v in contact_ref.values():
```

Comment:
Is this for multi chain support ? why did you change here as well the only_backbone_atom ?

Reply:
yes, for both multi-chain support and varied chain ID support.
The argument `return_only_backbone_atom` does not exist any more  as explained above. And backbone atoms will be got by using `get()` later in the code.